module/optee/voltd_regulator/include/mod_optee_voltd_regulator.h

```diff
@@ -7,31 +7,18 @@
 #ifndef MOD_OPTEE_VOLTD_REGULATOR_H
 #define MOD_OPTEE_VOLTD_REGULATOR_H
 
-#include <fwk_element.h>
-#include <fwk_macros.h>
-
-#include <stdint.h>
 #include <stdbool.h>
 
-struct scmi_server_regu_channel;
 struct regulator;
```

Comment:
I understand this opaque structure definition is coming from OP-TEE definitions, but is there a way to rename it? to be align with the rest name styles.

Reply:
Sorry, OP-TEE will not rename a struct because of SCP-fmw convention.
This is what we've done in the other _modules/optee/*_ modules.

Reply:
Sorry I meant to rename inside 'SCP-firmware' since it is only being used as a pointer casting to `struct regulator *` can be done inside `mod_optee_voltd_regulator_configuration.c` file instead.